packages/metals-vscode/src/debugger/scalaDebugger.ts

```diff
@@ -76,10 +76,10 @@ async function runMain(main: ExtendedScalaRunMain): Promise<boolean> {
       },
       {}
     );
-
+    const file = currentFile();
```

Comment:
This might prove problematic if we run but the file is not focused :thinking: 

Actually, we can already make it better via `main.class` which will be the name of the class being run, which should be unique.

Reply:
Good idea. testing it now.